CIPs/0010.md

```diff
@@ -0,0 +1,56 @@
+# CIP 0010: General Account Signer Authorization
+
+- Date: 2020-05-13
+- Author: @nambrot
+- Status: WIP
```

Comment:
Hi @nambrot as in the other PRs, please ensure the CIP is following the new template for the newly activated CIP process here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Please ensure it's in Draft status in order for me to also merge it in.